Resources/Locale/en-US/traits/traits.ftl

```diff
@@ -55,3 +55,10 @@ trait-name-Foreigner = Foreigner
 trait-description-Foreigner =
     For one reason or another you do not speak this station's primary language.
     Instead, you have a translator issued to you that only you can use.
+
+trait-name-ParkourTraining = Parkour Training
+trait-description-ParkourTraining =
+    Whether as a hobby, lifestyle, or professional training, you are trained in the discipline of parkour.
+    You climb 25% faster over tables, railings, fences and other similar structures.
+    You crawl 25% faster.
+    You can lie down and stand up in succession 10% faster.
```

Comment:
```suggestion
    You climb, crawl, and lie faster
```

Reply:
![image](https://github.com/user-attachments/assets/226cee76-fc7e-4fc6-8689-8b87fc1eae16)

Reply:
I forgor to make it not a draft

Reply:
Resolved with [75e35bc](https://github.com/Simple-Station/Einstein-Engines/pull/670/commits/75e35bc6190daa7d803efb25f20c26bb886f2741)

> Whether as a hobby, lifestyle, or professional training, you are trained in the discipline of parkour.
> You're faster with climbing, crawling, lying down, and getting up.